File: dart/lib/model/miner_miner_chain_info.dart
part of swagger.api;

class MinerMinerChainInfo {
  
  List<String> addresses = [];

  String owner = null;

  String peerId = null;

  String worker = null;

  MinerMinerChainInfo();

  @override
  String toString() {
    return 'MinerMinerChainInfo[addresses=$addresses, owner=$owner, peerId=$peerId, worker=$worker, ]';
  }

  MinerMinerChainInfo.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    addresses = (json['addresses'] as List).map((item) => item as String).toList();
    owner = json['owner'];
    peerId = json['peerId'];
    worker = json['worker'];
  }

  Map<String, dynamic> toJson() {
    return {
      'addresses': addresses,
      'owner': owner,
      'peerId': peerId,
      'worker': worker
     };
  }

  static List<MinerMinerChainInfo> listFromJson(List<dynamic> json) {
    return json == null ? new List<MinerMinerChainInfo>() : json.map((value) => new MinerMinerChainInfo.fromJson(value)).toList();
  }

  static Map<String, MinerMinerChainInfo> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, MinerMinerChainInfo>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new MinerMinerChainInfo.fromJson(value));
    }
    return map;
  }
}

File: csharp-dotnet2/src/main/CsharpDotNet2/estuary-client/Model/MinerMinerChainInfo.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class MinerMinerChainInfo {
    /// <summary>
    /// Gets or Sets Addresses
    /// </summary>
    [DataMember(Name="addresses", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "addresses")]
    public List<string> Addresses { get; set; }

    /// <summary>
    /// Gets or Sets Owner
    /// </summary>
    [DataMember(Name="owner", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "owner")]
    public string Owner { get; set; }

    /// <summary>
    /// Gets or Sets PeerId
    /// </summary>
    [DataMember(Name="peerId", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "peerId")]
    public string PeerId { get; set; }

    /// <summary>
    /// Gets or Sets Worker
    /// </summary>
    [DataMember(Name="worker", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "worker")]
    public string Worker { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class MinerMinerChainInfo {\n");
      sb.Append("  Addresses: ").Append(Addresses).Append("\n");
      sb.Append("  Owner: ").Append(Owner).Append("\n");
      sb.Append("  PeerId: ").Append(PeerId).Append("\n");
      sb.Append("  Worker: ").Append(Worker).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}
